src/main/java/org/manifold/compiler/front/ExpressionEdge.java

```diff
@@ -30,8 +33,7 @@ public void setTarget(ExpressionVertex newTarget) {
 
   public ExpressionEdge(ExpressionVertex from, ExpressionVertex to) {
     if (from == null && to == null) {
-      throw new UndefinedBehaviourError(
-          "attempt to create a totally disconnected ExpressionEdge");
+      log.warn("attempt to create a totally disconnected ExpressionEdge");
```

Comment:
Why did you change this to a `warn`? It seems like it would be hard to recover from this error.